metaconfig-core/shared/src/main/scala/metaconfig/internal/ConfGet.scala

```diff
@@ -20,10 +18,11 @@ object ConfGet {
     }
 
   def getOrElse[T](conf: Conf, default: T, path: String, extraNames: String*)(
-      implicit ev: ConfDecoder[T]
+      implicit ev: ConfDecoderWithDefaultMaybe[T]
```

Comment:
sorry. i was blind. i don't know why. ignore the last comment. brrr...

Reply:
:ok_hand: